posthog/models/special_migration.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+# an enum, essentially
+class MigrationStatus:
+    NotStarted = 0
+    Running = 1
+    CompletedSuccessfully = 2
+    Errored = 3
+    RolledBack = 4
+
+
+class SpecialMigration(models.Model):
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["name"], name="unique name",)]
```

Comment:
Do we need this?

Reply:
I believe it's good to have yes

Reply:
Can you explain why? Why do we need this additional constraint? Isn't the PK on `id` already enough?

Reply:
Essentially, I want to prevent at the db-level the case where we have 2 or more instances of a migration in the database, which can be at different statuses etc. 

We can ofc guard against this in code, but I think it's worth having the constraint. Although as I'm typing Im thinking we could get rid of ID altogether and make the name the PK?

Reply:
👍 ok if we think there's a possibility we might end up with duplicate migrations